test/mlir/onnx/parse/test_gridsample.json

```diff
@@ -0,0 +1,124 @@
+// RUN: sed 's/^[ \t]*\/\/.*//' %s | onnx-mlir --EmitONNXBasic --printIR 2> test_gridsample.log; cat test_gridsample.log | FileCheck --check-prefix=FAILED %s
```

Comment:
MLIR-Windows-CI seems to run in powershell and I couldn't come
up with a cross-platform way to pipe from a command with a
non-zero error code directly into FileCheck, so instead piping the
stderror output to a file and then piping that file into FileCheck

initially, I had copied the approach from `test/mlir/accelerators/nnpa/transform/zlow-normalize-by-using-dummyop.mlir` but that didn't work in powershell - I think we never saw that lit test fail the MLIR-Windows-CI because that CI isn't configured with `-DONNX_MLIR_ACCELERATORS=NNPA`

Reply:
This solution seems to work only when no --split-file option is in.

Reply:
right, it only works without --split-input-file

I think that's an ok compromise

as it happens, onnx-mlir doesn't support --split-input-file and all the parse lit tests can only do one onnx model per lit test - I don't see an easy way around that and I propose that we live with that limitation

Reply:
But if I understand well, now that onnx-mlir JSON input ignores the comment lines, we can again work normally.

Reply:
we still need the workaround (pipe to an error log, semicolon, and then cat the error log) to make the RUN command work cross-platform when onnx-mlir returns a non-zero return code, it's needed in the two lit tests that test that parsing fails as expected, namely `test_gridsample.json` and `zipmap.json`